eslintrc.json

```diff
@@ -219,6 +220,21 @@
       "functionPrototypeMethods": false
     }],
     "yoda": 0,
-    "sensible/check-require": [2, "always"]
+    "sensible/check-require": [2, "always"],
+    "logdna/grouped-require": 2,
+    "logdna/require-file-extension": 2,
+    "logdna/tap-consistent-assertions": [2, {
+      "preferredMap": {
+        "equal": "strictEqual",
+        "not": "notStrictEqual",
+        "same": "deepEqual",
+        "notSame": "notDeepEqual",
+        "strictSame": "strictDeepEqual",
+        "strictNotSame": "strictDeepNotEqual",
+        "match": "match",
+        "notMatch": "notMatch",
+        "type": "type"
```

Comment:
we'll want to decide on which synonyms we prefer here - this is just my initial take

reference: https://github.com/logdna/eslint-plugin-logdna/blob/master/lib/common/tap-synonyms.js#L29-L89

Reply:
If they are omitted, are they ignored?

Reply:
if not, `error: error`

Reply:
yes if there is no mapping, then nothing is done for that alias. I added `error: error` here.